src/Overview/ProjectLane/BranchCard/BranchCard.js

```diff
@@ -4,73 +4,85 @@ import ProgressLane from './CardLanes/ProgressLane/ProgressLane.js';
-          <a href={this.branch.getJiraLink()} className="white-text" target="_blank">
+          <a href={this.state.branch.getJiraLink()} className="white-text" target="_blank">
             #{this.state.issueNumber}
           </a>
         </span>
-      );
-    }
-    else {
-      return this.state.branchName;
+            );
+        }
+        else {
+            return this.state.branchName;
+        }
     }
-  }
 
-  getColorBasedOnLatestBuild() {
-    if(!this.branch.didTravisRun()) {
-      return "grey darken-2";
+    getColorBasedOnLatestBuild() {
+        if(!this.state.branch.didTravisRun()) {
+            return "grey darken-2";
+        }
+        else if(this.state.branch.didLatestBuildPass()) {
+            return "green darken-3";
+        }
+        else {
+            return "deep-orange";
+        }
     }
-    else if(this.branch.didLatestBuildPass()) {
-      return "green darken-3";
-    }
-    else {
-      return "deep-orange";
-    }
-  }
 
-  getColorBasedOnQualityGate() {
-    if(!this.branch.didSonarqubeRun()) {
-      return "grey darken-2";
-    }
-    else if(this.branch.doesPassQualityGate()) {
-      return "green darken-3";
+    getColorBasedOnQualityGate() {
+        if(!this.state.branch.didSonarqubeRun()) {
+            return "grey darken-2";
+        }
+        else if(this.state.branch.doesPassQualityGate()) {
+            return "green darken-3";
+        }
+        else {
+            return "deep-orange";
+        }
     }
-    else {
-      return "deep-orange";
-    }
-  }
 
-  render() {
-    return (
-      <div className="branch-card col s2" key={this.branch.getName()} >
-        <div className={`card small horizontal hoverable ${this.getColorBasedOnLatestBuild()}`}>
-          <div className="card-content white-text">
-            <span className="card-title">{this.getTitleForCard()}</span>
-            <div className="status-lanes">
-              <TextLane value={`</> ${this.branch.getNiceSonarqubeString()}`} color={this.getColorBasedOnQualityGate()} />
-              <ProgressLane value={this.branch.getCoverage()} valueText={this.branch.getNiceCoverage()} changeText={this.branch.getNiceCoverageChange()} showLabels={false} />
-              <UserLane username={this.branch.getLatestCommitter()} lastCommitTime={this.branch.getLastCommitTimeAsString()} avatarUrl={this.branch.getLatestCommitterAvatarUrl()}/>
+    render() {
+        return (
+            <div className="branch-card col s2" key={this.state.branchName} >
+                <div className={`card small horizontal hoverable ${this.getColorBasedOnLatestBuild()}`}>
+                    <div className="card-content white-text">
+                        <span className="card-title">{this.getTitleForCard()}</span>
+                        <div className="status-lanes">
+                            <TextLane value={`</> ${this.state.branch.getNiceSonarqubeString()}`} color={this.getColorBasedOnQualityGate()} />
```

Comment:
Please break this and the following 2 lines. 
Corresponding code convention: If a component initialises with 2 or more parameters, insert a line break before each parameter.